calculatorUICOOP/ViewModels/MainPageViewModel.cs

```diff
@@ -79,21 +83,29 @@ public void ShowNumberOnDisplay(string textToDisplay)
         public void ClearScreen()
         {
             DisplayContent = "0";
+            _hasDecimal = false;
         }
 
-        public void ShowPlusOnDisplay(string plus)
+        private void AssignOperator(string newOperator)
         {
+            _number1 = Convert.ToDouble(DisplayContent);
```

Comment:
Should be `decimal`. Since `MathLogic` class uses `double` a refactor story will be opened